shop/graphql/checkout/mutations.py

```diff
@@ -4,52 +4,59 @@
 from ...checkout.models import Checkout, CheckoutLine
 
 class CheckoutLineCreateInput(graphene.InputObjectType):
-    quantity = graphene.Int()
-    variant_id = graphene.Int()
+    quantity = graphene.Int(required=True)
+    variant_id = graphene.Int(required=True)
+    checkout_id = graphene.Int(required=False)
+
 
 class CheckoutCreateInput(graphene.InputObjectType):
-    user_email = graphene.String(required=True)
+    user_email = graphene.String(required=False)

```

Comment:
We should add some validation for fields `user_email` and `user`. We need to require one of these fields to track user.